geometry_msgs/Vector3 pos_sp
geometry_msgs/Vector3 vel_ff
float32 yaw_sp

